src/app/component/user/modal-auth/windowsignin/windowsignin.component.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, Inject, OnInit, ViewEncapsulation } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material';
+import { UserOwnSignInService } from '../../../../service/auth/user-own-sign-in.service';
+import { UserOwnSignIn } from '../../../../model/user-own-sign-in';
+import { HttpErrorResponse } from '@angular/common/http';
+import { UserSuccessSignIn } from '../../../../model/user-success-sign-in';
+import { Router } from '@angular/router';
+import { AuthService, GoogleLoginProvider } from 'angularx-social-login';
+import { GoogleSignInService } from '../../../../service/auth/google-sign-in.service';
+import { MatDialog } from '@angular/material';
+import { RestoreComponent } from '../../restore/restore.component';
+import { LocalStorageService } from '../../../../service/localstorage/local-storage.service';
+//import { WindowsignupComponent } from 'src/app/component/user/modal-auth/windowsignup/windowsignup.component';
+
+@Component({
+  selector: 'app-windowsignin',
+  templateUrl: './windowsignin.component.html',
+  styleUrls: ['./windowsignin.component.css']
+})
+export class WindowsigninComponent implements OnInit {
+  userOwnSignIn: UserOwnSignIn;
+  loadingAnim: boolean;
+
+  emailErrorMessageBackEnd: string;
+  passwordErrorMessageBackEnd: string;
+
+  backEndError: string;
+
+  constructor(
+  private matDialogRef: MatDialogRef<WindowsigninComponent>,
```

Comment:
Bad alignment of the constructor arguments. Please fix it

Reply:
ok